docs/dosbox.1

```diff
@@ -173,9 +172,6 @@ If you do specify a label this label will be kept as long as the drive
 is mounted. It will not be updated !!
 .RE
 .TP
-.B \-ioctl   
-Forces to use ioctl commands.
-.TP
 .BI \-usecd " number"
 Forces to use SDL cdrom support for drive number.
```

Comment:
Interesting that `-noioctl` isn't mentioned in this man-page.
I think adding it out-of-scope from the PR, and our trajectory toward dropping SDL CD-ROM support also doesn't warrant us bothering to keep track of this for future additions)